### PR TITLE
Various updates based on issues

### PR DESCRIPTION
**Pull request by dabit3**
Addresses following issues:

#2529 
#2543 
#2560 
#2517 
#2502 
#1188

### CODE REVIEW

File: docs/lib/auth/fragments/js/manageusers.md [hunk 1]
```diff
@@ -99,6 +100,14 @@ Auth.currentAuthenticatedUser({
 This method can be used to check if a user is logged in when the page is loaded. It will throw an error if there is no user logged in.
 This method should be called after the Auth module is configured or the user is logged in. To ensure that you can listen on the auth events `configured` or `signIn`. [Learn how to listen on auth events.](~/lib/utilities/hub.md#authentication-events)
 
+### Retrieve attributes for current authenticated user
+
+You can also access the user's attributes like their email address, phone number, sub, or any other attributes that are associated with them from the user object returned by `Auth.currentAuthenticatedUser`.
```
Comment (mauerbac) [inline docs/lib/auth/fragments/js/manageusers.md:105]: nit: what is sub here?

**Review by mauerbac — APPROVED** [review]
lgtm, will wait for another review before merging

File: docs/lib/auth/fragments/js/manageusers.md (hunk 1 above)
Comment (amhinson) [reply docs/lib/auth/fragments/js/manageusers.md:105]: Short for subject (OIDC spec)

**Review by amhinson — APPROVED** [review]

File: docs/start/getting-started/fragments/reactnative/setup.md [hunk 2]
```diff
@@ -10,6 +10,8 @@ To get started, initialize a new React Native project.
 ```bash
 npm install -g expo-cli  
 expo init RNAmplify
+
+? Choose a template: blank
```
Comment (ericclemmons) [inline docs/start/getting-started/fragments/reactnative/setup.md:14]: Thanks for this!  A small tweak, but helps reduce confusion.

**Review by ericclemmons — APPROVED** [review]
Thanks for these fixes!

File: docs/start/getting-started/fragments/angular/auth.md [hunk 3]
```diff
@@ -45,7 +45,7 @@ Add the `amplify-authenticator` component to the top of `src/app/app.component.h
 Run the app to see the new Authentication flow protecting the app:
 
 ```bash
-ng serve
+npm start
```
Comment (undefobj) [inline docs/start/getting-started/fragments/angular/auth.md:48]: Nit: These things are *basically* the same now that Angular CLI uses webpack but they used to be different. For older Angular CLIs this won't be the same result.

**Review by undefobj — APPROVED** [review]
One minor nit but LGTM.

**Review by yuth — APPROVED** [review]
LGTM!

**Pull request merged**